code/game/objects/effects/aliens.dm

```diff
@@ -193,49 +193,58 @@
 	fire_level_to_extinguish = 6
 	time_to_live = 6
 
-	var/bonus_damage = 5
-
-/obj/effect/xenomorph/spray/strong
-	name = "strong splatter"
-	desc = "It burns a lot!"
-	icon_state = "acid2-strong"
-
-	stun_duration = 2
-	damage_amount = 30
-	fire_level_to_extinguish = 18
-	time_to_live = 3 SECONDS
-	// Stuns for 2 seconds, lives for 3 seconds. Seems to stun longer than it lives for at 2 seconds
-
-/obj/effect/xenomorph/spray/strong/no_stun
-	stun_duration = 0
+	var/bonus_damage = 25
 
 /obj/effect/xenomorph/spray/weak/apply_spray(mob/living/carbon/M)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 
 		var/damage = damage_amount
 
-		var/should_stun = FALSE
-		for (var/datum/effects/acid/A in H.effects_list)
-			should_stun = TRUE
-			damage += (-1*(A.duration - A.original_duration))*(A.damage_in_total_human/A.original_duration)
+		var/buffed_splash = FALSE
+		var/datum/effects/acid/A = locate() in H.effects_list
+		if(A && A.acid_enhanced == FALSE) // can't stack the bonus every splash. thatd be nuts!
+			buffed_splash = TRUE
```

Comment:
you touched too much of this code (it appears entirely in green) to not have to change this var name

Reply:
don